src/DroneSafetyApi/Models/Heatmap.cs

```diff
@@ -1,12 +1,81 @@
-﻿using System;
+﻿using Newtonsoft.Json;
+using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text;
 using System.Threading.Tasks;
 
 namespace DroneSafetyApi.Models
 {
-    public class Heatmap
+
+    public class HeatMap
     {
-        public int[][] Values { get; set; }
+        private int[,] mHeatMap;
```

Comment:
It's not happening anywhere else. Also, this notation is redundant with things like properties, it doesn't help